terraform/modules/locality_kubernetes/kubernetes.tf

```diff
@@ -0,0 +1,110 @@
+variable "environment" {
```

Comment:
Could you please move the other per-locality Kubernetes resources created in `main.tf` into this module? `resource "kubernetes_namespace" "namespaces"` and `resource "kubernetes_secret" "ingestion_packet_decryption_keys"` would fit in here well. The values that are then needed in `data_share_processors.tf` can be exposed from this module as output variables.

Reply:
Will address this in a future PR.